test/t8n/t8n.cpp

```diff
@@ -54,6 +55,13 @@ int main(int argc, const char* argv[])
             output_result_file = argv[i];
         else if (arg == "--output.alloc" && ++i < argc)
             output_alloc_file = argv[i];
+        else if (arg == "--state.reward" && ++i < argc)
+        {
+            if (std::string(argv[i]) == "-1")
+                state_reward = std::numeric_limits<intx::uint256>::max();
```

Comment:
This is for disabling it? Why not use `std::optional<intx::uint256>` in that case?

Reply:
I agree here.